baybe/acquisition/acqfs.py

```diff
@@ -120,6 +120,21 @@ def get_integration_points(self, searchspace: SearchSpace) -> pd.DataFrame:
         return result
 
 
+########################################################################################
+### Knowledge Gradient
+@define(frozen=True)
+class qKnowledgeGradient(AcquisitionFunction):
+    """Monte Carlo based knowledge gradient."""
```

Comment:
I would already add a comment here that this is only applicable in continuous search spaces.

Reply:
Good idea. And perhaps: if we really want to dive into it, we could think about if its worth figuring out why it's restricted in the first place (i.e., I still see no conceptual reason, so either I'm overlooking some critical piece or the discrete implementation is simply missing) and open a corresponding botorch issue? Not sure if anyone has time for the investigation at the moment though, given that we have so many open work streams

Reply:
text added

comment added to pour excel file to revisit discrete issue with botorch